values/teal-x86_64.yaml

```diff
@@ -4,7 +4,7 @@ codename: "teal"
 arch: "x86_64"
 golang_arch: "amd64"
 
-tool_image: opensuse/leap:15.3
```

Comment:
I'd be tempted to stay stable on teal, and avoid tw or latest tags. We can use the other flavors to assess issues with newer images

Reply:
done!